tools/toolchain/scripts/install_fftw.sh

```diff
@@ -19,10 +19,11 @@ FFTW_LIBS_OMP=''
 cd "${BUILDDIR}"
 case "$with_fftw" in
     __INSTALL__)
+        require_env MPI_LIBS
         echo "==================== Installing FFTW ===================="
         pkg_install_dir="${INSTALLDIR}/fftw-${fftw_ver}"
         install_lock_file="$pkg_install_dir/install_successful"
-        if verify_checksums "${install_lock_file}" ; then
+        if [[ $install_lock_file -nt $SCRIPT_NAME ]]; then
```

Comment:
We are now using `verify_checksums` instead of timestamps to check if a package needs to be re-installed. This is a preparation for having better dependency tracking in the toolchain.